src/main/java/com/powsybl/openloadflow/network/ReactivePowerControl.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network;
+
+import java.util.*;
+
+/**
+ * @author Bertrand Rix <bertrand.rix at artelys.com>
+ */
+public class ReactivePowerControl {
+
+    private final LfBus controlled;
+
+    private final Set<LfBus> controllers;
+
+    private final double targetValue;
+
+    public ReactivePowerControl(LfBus controlled, double targetValue) {
+        this.controlled = controlled;
+        this.targetValue = targetValue;
+        this.controllers = new LinkedHashSet<>();
+    }
+
+    public double getTargetValue() {
+        return targetValue;
+    }
+
+    public LfBus getControlledBus() {
```

Comment:
You should have a `getControlledBranch`.

Reply:
Yes this is still basically a copy of VoltageControl class. I will update that with the new information you have given me regarding DiscretePhaseControl class.